plugins/slack_app/README.md

```diff
@@ -0,0 +1,12 @@
+# Slack App Plugin
+
+This plugin adds a `/deploy` command to Slack which creates Samson deployments.
+To activate:
+
+1. Go over to Slack and create a [new app](https://api.slack.com/apps/new). Many of the fields are completely up to you, but these are important:
+  - Grab the Client ID and secret and verification token, and record them in your environment as `SLACK_CLIENT_ID`, `SLACK_CLIENT_SECRET`, and `SLACK_VERIFICATION_TOKEN`.
```

Comment:
I think it's good idea to update the README and move the credentials part right before `2.`